src/main/steps/applicant/statement-of-service/view-respondant-documents/content.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  ApplicationPackDocumentMeta,
+  CitizenRespondentPack,
+} from '../../../../../main/steps/common/documents/definitions';
+import { TranslationFn } from '../../../../app/controller/GetController';
+import { FormContent } from '../../../../app/form/Form';
+import { applyParms } from '../../../../steps/common/url-parser';
+import { DOWNLOAD_DOCUMENT } from '../../../../steps/urls';
+
+export const en = {};
+
+export const cy = {};
+
+const languages = {
+  en,
+  cy,
+};
+
+export const form: FormContent = {
+  fields: {},
+};
+
+export const generateContent: TranslationFn = content => {
+  const translations = languages[content.language];
+  let documents;
+  const respondentPacks: CitizenRespondentPack =
+    content.additionalData?.req.session.userCase.personalServiceUnServedRespondentPack;
+  if (respondentPacks) {
```

Comment:
what happens when respondentPacks is empty? what is document var set to? Unnecessary declaration of documents and respondentPacks. Use ES6 conditional operator and coalescing to return documents directly from the return statement.